relay-server/src/services/processor.rs

```diff
@@ -1095,7 +1100,11 @@ impl EnvelopeProcessorService {
             };
 
             metric!(timer(RelayTimers::EventProcessingLightNormalization), {
-                normalize_event(event, &config).map_err(|_| ProcessingError::InvalidTransaction)
+                validate_transaction(event, &tx_validation_config)?;
+                validate_event_timestamps(event, &event_validation_config)
+                    .map_err(|_| ProcessingError::InvalidTransaction)?;
+                normalize_event(event, &normalization_config);
+                Result::<(), ProcessingError>::Ok(())
```

Comment:
`Ok(())` isn't sufficiently explicit.

Reply:
Why?

Reply:
The LSP doesn't recognize what type of error the `Result` is, and being explicit makes it happy.